.github/workflows/storyblock.yml

```diff
@@ -0,0 +1,32 @@
+name: Storyblock CI
```

Comment:
❓ is StoryBlock one word (as here) or two (as readme), maybe you should add a `-` or `_` everywhere